_tips/writing.id.md

```diff
@@ -0,0 +1,470 @@
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Gunakan judul untuk menyampaikan makna dan struktur
+
+Gunakan judul pendek untuk mengelompokkan paragraf terkait dan menerangkan bagian-bagiannya dengan jelas. Judul yang baik menggambarkan garis besar isinya.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Gunakan judul untuk mengelompokkan konten" class="example" %}
+{:/}
+
+<div class="heading-structure two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Salah" %} Kekurangan judul</figcaption>
+    <div>
+      {% include_cached img.html src="headings-poor.png" alt="Contoh tidak ada judul, lihat di bawah untuk contoh mendetail" %}
+    </div>
+{% include_cached excol.html type="start" id="no-headings" %}
+Lihat contoh *inline*
+{% include_cached excol.html type="middle" %}
+<h2 id="no-headings-modal-title"><span class="visuallyhidden">Contoh: </span>Judul dan Subjudul</h2>
+
+<p>Elemen HTML menyediakan informasi hierarki struktural suatu dokumen. Penggunaan elemen yang benar akan membantu menyampaikan makna tambahan pada teknologi pendukung. Dalam banyak kasus, hal ini juga akan membuat dokumen Anda lebih mudah diedit.</p>
+
+<p>Untuk dokumen yang panjangnya lebih dari tiga atau empat paragraf, judul dan subjudul penting untuk ketergunaan dan aksesibilitas. Mereka membantu pembaca dalam menentukan garis besar isi dokumen dan menavigasi ke informasi spesifik yang dicari.</p>
+
+<p>Judul diklasifikasikan ke dalam tingkatan dari satu hingga enam. Tingkat tertinggi adalah "Level 1" dan sering kali berhubungan dengan judul halaman atau bagian utama dari dokumen.</p>
+
+<p>Pembaca visual mengidentifikasi judul dengan memindai halaman untuk mencari teks dengan ukuran yang lebih besar atau gaya yang berbeda. Pengguna teknologi pendukung tidak dapat melihat perubahan visual tersebut, sehingga mengubah gaya saja tidak cukup untuk menjadi petunjuk.</p>
+
+<p>Alhasil, judul harus "ditandai" secara semantik agar teknologi pendukung dapat mengidentifikasi judul. Kemudian judul juga akan bisa digunakan untuk menavigasi.</p>
+
+<p>Hal ini menjadikan penambahan judul salah satu sarana terpenting bagi pengguna pembaca layar, sehingga mereka dapat mempelajari apa yang ada pada halaman. Perhatikan bahwa pemberian tanda biasanya memicu perubahan format secara visual yang bisa disesuaikan pada banyak dokumen.</p>
+
+<cite>Diadopsi dari <a href="http://accessibility.psu.edu/headings/">"Judul dan Subjudul" di Penn State</a></cite>
+{% include_cached excol.html type="end" %}
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="Oke" %} Menggunakan judul dan subjudul</figcaption>
+    <div>
+      {% include_cached img.html src="headings-good.png" alt="Ilustrasi struktur judul yang bagi, lihat di bawah untuk contoh mendetail"%}
+    </div>
+{% include_cached excol.html type="start" id="headings-good" %}
+Lihat comtoh *inline*
```

Comment:
@frirasyidi If "inline" is an anglicism, would you wrap it with `<span lang"en"></span>`?

Reply:
@remibetin Fixed on the next commit.